### ISSUE TITLE
Azure Functions Python Runtime 08/16/2019 [breaking]

### ISSUE THREAD

**Issue by maiqbal11** [Azure/azure-functions-python-worker#521]
**TL;DR**

You will need to make the following changes to your Python Function App to avoid breaking:

1. Update `azure-functions` to the latest in your `requirements.txt`. Either set `azure-functions>=1.0.0b5` or simply include `azure-functions` (always pulls latest). If it is not present, please add it in. 
2. Remove `azure-functions-worker` from your `requirements.txt` if present.
3. Republish your function app.

**What's changing?**
 
1. We are making changes to our Python app runtime flow to remove collisions between the user's requirements and those required for our runtime. For this, we have removed `azure-functions-worker` from the `azure` namespace and to its own. 
2. `azure-functions` is moving towards being a user controlled dependency rather than one that is managed by us. Versions greater than `1.0.0b5` are compatible with the latest changes. 

**Rationale**

1. `azure-functions` is a user facing package that lives in the `azure` namespace. We want users to be able to manage it as such. In this release, we are decoupling the dependencies between the `azure-functions-worker` and `azure-functions`. In the future, we can release new major versions of the `azure-functions` library without breaking existing customers since our runtime only relies on `azure-functions-worker`. Users will have the choice of which `azure-functions` library version to consume. 

2. During the preview time period, we saw instances where the users' including an `azure` package in their `requirements.txt` (e.g. `azure-cosmos`) led to import failures for the user. The reason for this was that the `azure` namespace for our system dependencies (i.e. `azure.functions_worker`) was not consistent with other `azure` namespaces (in a nutshell, some `azure.xxx` packages included `__init__.py` files and others did not). This meant that imports from the user's namespace (which lives in a separate directory) could not be resolved because `azure.xxx` modules were only searched for in our namespace. Thus we made the decision to remove the `azure-functions-worker` from the `azure` namespace altogether and will be managing it ourselves from now on. 

Please use https://github.com/Azure/app-service-announcements-discussions/issues/111 for discussion.

**Comment by anirudhgarg** [comment]
Closing this issue, still keeping it pinned.
